src/backend/newman.c

```diff
@@ -1219,7 +1213,6 @@ STATIC void cpp_argument_list(type *t, int flag)
         {
             if (i == mangle.argi)               // no match
             {
-#if OVERLOAD_CV_PARAM
```

Comment:
Apparently in master the definition still exists (https://github.com/D-Programming-Language/dmd/blob/master/src/backend/cdef.h#L264) but the two uses in this file are already gone.  Maybe you rebased against an old master?